src/app/page.js

```diff
@@ -0,0 +1,74 @@
+"use client";
+
+import { Card, Text, Subtitle, Title, BarChart } from '@tremor/react';
+import React, { useState, useEffect } from 'react';
+
+// Get your Tinybird host and token from the .env file
+const TINYBIRD_HOST = process.env.NEXT_PUBLIC_TINYBIRD_HOST; // The host URL for the Tinybird API
+const TINYBIRD_TOKEN = process.env.NEXT_PUBLIC_TINYBIRD_TOKEN; // The access token for authentication with the Tinybird API
+
+export default function Dashboard() {
+    // React state hook for managing the "newSignaturesPerDay" data
+    // Initializes data with an array containing an object with default values for the top orgs creating signatures
+    const [ranking_of_top_organizations_creating_signatures, setData] = useState([{
+        "organization": "",
+        "org_total": 0,
+    }]);
+   
+    // Initializes latency with an integer 0  
+    const [latency, setLatency] = useState(0);
+
+    // Define hardcoded date range for the query
```

Comment:
@juliavallina here's the section we worked on

Reply:
Ready!